resources/texts/_contentManager.html

```diff
@@ -10,30 +10,30 @@
 <div id="app">
   <div id="searchBar">
     <form>
-      <input id="searchInput" type="text" placeholder="Search files" oninput="setSearch(this.value)"/>
+      <input id="searchInput" type="text" :placeholder="translations['search-files']" oninput="setSearch(this.value)"/>
```

Comment:
It would be better to have a function (`gt` as on Qt side) to do the lookup for us.
Then we would have to write `gt('search-files')` only (a bit smaller)

Reply:
I prefer to use a map which is filled at the initialisation because using a function for each text to translate that will call  a c++ method (each call is asynchrone) is much more complicated than only one call at the beginning I think

Reply:
I have the idea that the function do the lookup in the map for us.
It is just syntactic sugar. Using a map is the good thing to do.